Zend/zend.c

```diff
@@ -1503,6 +1503,103 @@ ZEND_API ZEND_COLD ZEND_NORETURN void zend_error_at_noreturn(
 	abort();
 }
 
+static ZEND_COLD void ZEND_FASTCALL zend_function_error_noreturn_variadic(
+	int orig_type, const char *error_filename, uint32_t error_lineno, const char *function, const char *format, va_list args
+) {
+	zend_op_array *op_array = CG(active_op_array);
```

Comment:
I'm not sure if retrieving the current scope and function is a good idea. Probably I should pass them to the function directly?